### PR TITLE
fix(server): add password strength validation in user service

### PR DESCRIPTION
**Pull request by Mackbellemore**
Were gonna have client side validation on the password strength when we do the register page too

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-mgzq2p Password validation is wonky](https://app.clickup.com/t/mgzq2p)

**Comment by gordonpn** [comment]
I thought the issue was that we were validating the salted password?

**Review by KarinKazarian — APPROVED** [review]

File: server/app/models/UserModel.ts [hunk 1]
```diff
@@ -24,7 +24,6 @@ export const UserSchema: Schema = new Schema({
   password: {
     type: Schema.Types.String,
     required: true,
-    validate: [validator.isStrongPassword, 'Password must contain a capital, symbol and a number.'],
```
Comment (nasjuice) [inline server/app/models/UserModel.ts:27]: is this one brokey or we prefer to validate in the service.

**Comment by gordonpn** [comment]
Let’s not merge this one until we figure out what’s wrong with Zeet

File: server/app/models/UserModel.ts (hunk 1 above)
Comment (Mackbellemore) [reply server/app/models/UserModel.ts:27]: nah i was just being stupid, the password is hashed before the model validation :facepalm: So i need to validate before hashing in the service layer

**Pull request closed without merging**